Cairo Desktop/CairoDesktop.Common/Settings.cs

```diff
@@ -370,6 +370,20 @@ public bool EnableMenuBar
             set => Set(ref _enableMenuBar, value);
         }
 
+        private bool _enablePlacesOption = true;
+        public bool EnablePlacesOption
+        {
+            get => _enablePlacesOption;
+            set => Set(ref _enablePlacesOption, value);
+        }
+
+        private bool _enableProgramsOption = true;
+        public bool EnableProgramsOption
+        {
+            get => _enableProgramsOption;
+            set => Set(ref _enableProgramsOption, value);
+        }
+
```

Comment:
Could you please rename these settings to say `Menu` instead of `Option` (`EnablePlacesMenu` and `EnableProgramsMenu`)?

Reply:
Done. 👍